core/src/main/java/com/linecorp/armeria/common/logging/DefaultRequestLog.java

```diff
@@ -186,6 +189,11 @@ private void endResponseFrom(RequestLog log) {
         }
     }
 
+    @Override
+    public List<RequestLog> children() {
+        return children != null ? ImmutableList.copyOf(children) : Collections.emptyList();
```

Comment:
Recommend `ImmutableList.of()` over `Collections.emptyList()`, `ImmutableList.copyOf` is aware of the former. Small difference but may as well take advantage.